src/Stateless/Reflection/StateInfo.cs

```diff
@@ -143,7 +143,14 @@ public class StateInfo
         /// <summary> 
         /// Transitions defined for this state.
         /// </summary>
-        public IEnumerable<TransitionInfo> Transitions { get { return FixedTransitions.Concat<TransitionInfo>(DynamicTransitions); } }
+        public IEnumerable<TransitionInfo> Transitions 
+        {
+            get { 
+                return FixedTransitions == null // A quick way to check if AddRelationships has been called.
+                            ? null 
```

Comment:
Why not use null coalescing like: `FixedTransitions?.Concat<TransitionInfo>(DynamicTransitions)`

Reply:
I've not contributed to this project before and was a little worried about what version of c# it was safe to use. My Rider kept suggesting all sorts of refactorings, pattern matching etc. So simply playing it safe.

Reply:
Fair point, I believe it's C# 10 (the .NET 6.0 default) but there's no lang version set in the csproj. This is very readable so I think it's fine to leave as is.